.github/FUNDING.yml

```
@@ -0,0 +1,8 @@
+# These are supported funding model platforms
+
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
```

We're `nusmods` on Patreon https://www.patreon.com/nusmods

```suggestion
patreon: nusmods
```